asg.tf

```diff
@@ -9,6 +9,13 @@ resource "aws_security_group" "jenkins-master" {
     security_groups = ["${aws_security_group.jenkins-master-lb.id}"]
   }
 
+  ingress {
```

Comment:
I don't think this should be open to anything but the ALB. I can't see anything using it in the rest of the code, correct me if I'm wrong

Reply:
Same case as above. But as I updated the code I'm using sg instead of CIDR.